frontend/src/components/Navbar.jsx

```diff
@@ -117,7 +116,7 @@ function ResponsiveAppBar() {
                         sx={{ display: { xs: 'flex', md: 'none' }, mr: 1 }}
                     />
                     <Typography
-                        variant="h5"
+                        variant="h7"
```

Comment:
```suggestion
                        variant="h6"
```

`<h7>` [does not exist, ](https://mui.com/material-ui/react-typography/). Would `h6` work?